policy/centos9/scripts/sign

```diff
@@ -1,45 +1,40 @@
 #!/bin/bash
 set -e -x
```

Comment:
I wonder if it's best to remove `-x` to avoid having to set `+x` and `-x` multiple times in the script.